tests/rustdoc-gui/search-result-color.goml

```diff
@@ -28,6 +28,11 @@ define-function: (
             ".result-" + |result_kind| + ":focus ." + |result_kind|,
             {"color": |hover_color|},
         )
+        assert-css: (
+            ".result-name .typename",
+            {"color": |typename_color|},
```

Comment:
Since it's always the same, you can simply write:

```suggestion
            {"color": "#888"},
```

And then you can remove the newly added `store-value`.

Reply:
done